scripts/aws/monitor.py

```diff
@@ -1,6 +1,11 @@
 #!/usr/bin/python
 #
+# Pre-reqs:
 # Run this script from the directory its in
+# Make sure the API Gateways deployed have different names
```

Comment:
This isn't easy to control, especially with the current AWS API Gateway `ImportAPI` operation that we use to transform Swagger documents into API Gateway instances. `ImportAPI` forces the API name to match the `info.title` field in Swagger (which for us is "Open edX" in swagger/api.yaml). There's currently no way to override this; I suppose one could add a step during the upload phase to rename the gateway just after each Swagger upload, but that's not something we do today.